specs/RestartRequested.md

```diff
@@ -98,10 +175,19 @@ s
 ```c#
 namespace Microsoft.Web.WebView2.Core
 {
+    enum CoreWebView2RestartPriorityLevel
+    {
+        BestEffort = 0,
+        Critical = 1,
+    };
+
     runtimeclass CoreWebView2Environment
     {
         // ...
         event Windows.Foundation.TypedEventHandler<CoreWebView2Environment, Object> RestartRequested;
+
+        // Set restart priority level
+        CoreWebView2RestartPriorityLevel RestartPriorityLevel { get; set; };
```

Comment:
Can we call the enum 'CoreWebView2RestartRequestedPriority' and name the property just 'Priority'? We don't need to say 'Restart' in the property name if its on the RestartRequestedEventArgs since then Restart is in the class name already. And 'Level' is sort of unnecessary